lib/AST/ASTContext.cpp

```diff
@@ -952,7 +952,11 @@ ASTContext::getPointerPointeePropertyDecl(PointerTypeKind ptrKind) const {
   llvm_unreachable("bad pointer kind");
 }
 
-CanType ASTContext::getAnyObjectType() const {
+CanType ASTContext::getAnyExistentialType() const {
+  return ExistentialType::get(TheAnyType)->getCanonicalType();
```

Comment:
This should already be canonical, you can do `return CanType(ExistentialType::get(TheAnyType))` instead (it will assert if its not canonical)